app/scripts/directives/overview/listRow.js

```diff
@@ -158,9 +159,14 @@ function OverviewListRow($filter,
         return true;
       }
       // Start pipeline or build is displayed.
-      if ((_.size(row.buildConfigs) === 1 || _.size(row.pipelines) === 1) &&
-           canI('buildconfigs/instantiate')) {
-        return true;
+      if (canI('buildconfigs/instantiate', 'create')) {
```

Comment:
we are duplicating a lot of logic from showStartBuildAction and showStartPipelineAction, is there a reason not to just make calls to each of those?